addons/website_slides_survey/views/website_profile.xml

```diff
@@ -9,6 +9,59 @@
                 </div>
             </t>
         </xpath>
+        <!-- Certification Attempts -->
+        <xpath expr="//ul[hasclass('o_wprofile_nav_tabs')]" position="inside">
+            <li t-if="show_certification_tab" class="nav-item">
+                <a role="tab" aria-controls="certification" href="#profile_tab_content_certification" t-attf-class="nav-link #{ 'active' if active_tab == 'certification' else '' }" data-toggle="tab">
+                    <t t-if="certification_attempts" t-out="len(certification_attempts)" /> Certification Attempts
+                </a>
+            </li>
+        </xpath>
+        <xpath expr="//div[hasclass('o_wprofile_tabs_content')]" position="inside">
+            <div t-if="show_certification_tab" role="tabpanel" t-attf-class="tab-pane #{ 'show active' if active_tab == 'certification' else '' }" id="profile_tab_content_certification">
+                <h5 class="d-flex justify-content-between align-items-end border-bottom pb-1">
+                    Certification Attempts
+                    <form method="get">
+                        <div class="input-group" role="search">
+                            <input type="text" class="form-control" name="search" t-att-value="certification_search_terms or ''" placeholder="Search Attempts..."/>
```

Comment:
We could replace `name="search"` by something namespaced for certifications search, your test in the controller would make more sense then.

Reply:
I will replace "search" with "certification", the url will now look like that: `/profile/user/2?certification=Furniture+Creation+Certification` **Do you a better name ?**